src/rust/engine/engine_cffi/src/lib.rs

```diff
@@ -593,12 +594,29 @@ fn workunit_to_py_value(workunit: &Workunit) -> Option<Value> {
     ));
   }
 
```

Comment:
I think that we should assume that we'll end up with multiple artifact types attached to some workunits, so I'd suggest nesting them now under something called "artifacts" or "outputs" or something:
```
"artifacts": {
  "stdout": ..,
  "stderr": ..,
}
```